src/styles/components/modal/link-list/styles.less

```diff
@@ -0,0 +1,47 @@
+.modal-link-list {
```

Comment:
Why are these styles in `dcos-ui`, but the modal in private plugins?

Reply:
Removed